apps/payments/next/components/HelloWorld/index.tsx

```diff
@@ -0,0 +1,37 @@
+const HelloWorld = () => {
```

Comment:
Let's add examples that show styles being used that are specific to payments-next and also from shared-styles.

Reply:
I don't believe this was explicitly asked in the AC. Would this not push it out of scope? Should it maybe be a Polish PR instead?

Reply:
I agree with @xlisachan we should be adding some examples. Should be a quick change.

Just an FYI, the 3rd point in the AC does ask that the PR ensures the shared tailwind styles work.

> 3. Shared Tailwind styles can be used in payments-next and works with hot-reloading

Reply:
Roger that! 🖖 I'll delete all the boilerplate code as requested in the other comment and redo `HelloWorld`! For clarity, as I am a little confused and would like your guidance, by "examples" do you mean multiple components with stories (i.e. more than just `HelloWorld`)? Or did you want me to pass in styles to show different variations of `HelloWorld`? And should I still do `HelloWorld` if we're trying to keep it clean (i.e. should I mock one of the existing components)? I wasn't quite sure and want to make sure I'm doing it the way you intended. 🙏 TYIA!